src/main/java/ch/njol/skript/patterns/SkriptPattern.java

```diff
@@ -68,24 +67,6 @@ public String toString() {
 		return first.toFullString();
 	}
 
-	public static String[] getKeywords(PatternElement first) {
```

Comment:
We could keep this method with a deprecation notice _just in case_ somebody is still using it.